changelogs/fragments/9554-add-cpanm-option_with-recommends-and-suggests.yml

```
@@ -0,0 +1,3 @@
+minor_changes:
+  - cpanm - enable usage of option ``--with-recommends`` (https://github.com/ansible-collections/community.general/issues/9554, https://github.com/ansible-collections/community.general/pull/9555).
+  - cpanm - enable usage of option ``--with-suggests`` (https://github.com/ansible-collections/community.general/issues/9554, https://github.com/ansible-collections/community.general/pull/9555).
```

The issue is just about `--with-recommends`, not about `--with-suggests`:
```suggestion
  - cpanm - enable usage of option ``--with-suggests`` (https://github.com/ansible-collections/community.general/pull/9555).
```

---

Indeed, not link to any issue for `--with-suggests`. Cheers

---

@felixfontein should be ok now.